injective/assetlist.json

```diff
@@ -484,6 +484,131 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/astro.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/astro.svg"
       }
+    },
+    {
+      "description": "A receipt token for lent INJ issued by the Neptune Protocol.",
+      "denom_units": [
+        {
+          "denom": "inj1rmzufd7h09sqfrre5dtvu5d09ta7c0t4jzkr2f",
+          "exponent": 0
+        },
+        {
+          "denom": "nINJ",
+          "exponent": 18
+        }
+      ],
+      "base": "inj1rmzufd7h09sqfrre5dtvu5d09ta7c0t4jzkr2f",
+      "name": "Neptune Receipt INJ",
```

Comment:
How are these linked to the assets they are a receipt for? E.g., how is Neptune Receipt INJ related to INJ? I ask for clarity because these likely should all include the `traces` property to show their relation to their underlying asset.

Reply:
the Neptune Receipt INJ is provided as a redemption token for the deposit of INJ as a lending asset within the Neptune borrowing and lending protocol.

Reply:
Apologies for the delay.
I'm thinking this should probably be marked as a 'synthetic' of the asset they are a receipt for.
So, e.g., Neptune Receipt INJ would include:
```
"traces": [
  {
    "type": "synthetic",
    "counterparty": {
      "chain_name": "injective",
      "base_denom": "inj"
    },
    "provider": "Neptune"
  }
],
```
I hope this makes sense

Reply:
Can I bump this. Any objection to indicating these as synthetics? Or should this be closed?

Reply:
@JeremyParish69 The tokens are not 1:1 with the underlying asset. Their redemption ratio increases as interest is paid to lenders. I don't see any docs describing the exact definition of what "synthetic" means in this context.